api/app.py

```diff
@@ -40,7 +41,9 @@
 import routes
 import admin.routes
 
-debug = Configuration.logging_policy().get("level") == 'DEBUG'
+log_level = LogConfiguration.initialize(_db, testing=testing)
```

Comment:
I don't really like calling `initialize` again just to get the current log level, but I don't think it hurts performance all that much.